components/status_led.rst

```diff
@@ -9,7 +9,8 @@ The ``status_led`` hooks into all ESPHome components and can indicate the status
 the device. Specifically, it will:
 
 - Blink slowly (about every second) when a **warning** is active. Warnings are active when for
-  example reading a sensor value fails temporarily or the WiFi/MQTT connections are disrupted.
+  example reading a sensor value fails temporarily, the WiFi/MQTT connections are disrupted, or 
+  if the native API component is included but no client is connected.
```

Comment:
Shouldn't be changing unrelated files.

Reply:
Huh, how did that happen?

Reply:
Fixed

Reply:
> Huh, how did that happen?

Probably auto-formatting in your editor. Good practice before pushing a PR:

```
git pull upstream dev   (or next as appropriate)
git diff upstream/dev --name-only
```
And check that no files are listed as changed other than the intended ones.

Then after creating the PR do your own review in Github, before anyone else does.